src/main/java/dev/codesoapbox/dummy4j/dummies/shared/string/StringSanitizer.java

```diff
@@ -1,14 +1,14 @@
-package dev.codesoapbox.dummy4j.dummies.internet;
+package dev.codesoapbox.dummy4j.dummies.shared.string;
 
 import java.text.Normalizer;
 import java.util.regex.Pattern;
 
 /**
  * Provides methods for normalizing strings and removing unwanted characters from them
  *
- * @since 0.5.0
+ * @since SNAPSHOT
  */
-final class StringSanitizer {
+public final class StringSanitizer {
 
     /**
```

Comment:
I think this part:
```
* @see <a href="https://stackoverflow.com/questions/56653323/s-doesnt-actually-capture-all-whitespace-characters/
* 56654027#56654027">Capture all whitespace characters (StackOverflow)</a>
```
Is the reason why javadoc is currently failing.

The same problem is probably in `CreditCardNumberBuilder`.

The failing method is:
`mvn verify org.sonarsource.scanner.maven:sonar-maven-plugin:sonar -P code-coverage -P sonar-cloud`